### ISSUE TITLE
Get-NSXFirewallRule failure when specifying RuleId

### ISSUE THREAD

**Issue by dcoghlan** [vmware/powernsx#173]
Running the Get-NsxFirewallRule cmdlet to retrieve a specific individual firewall rule by RuleId produces the error below.
Current testbed is NSX-v 6.2.5. Error occurs for both local/global and universal rules.

Get-NsxFirewallRule -RuleId 2147483802

```                                                                                                      
The property 'firewallConfiguration' cannot be found on this object. Verify that the property exists.
At /Users/powernsx/PowerNSX.psm1:22449 char:17
+             if ($response.firewallConfiguration) {
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict
 ```

XML Response from NSX Manager when using the API directly.
```
<?xml version="1.0" encoding="UTF-8"?>
<filteredfirewallConfiguration timestamp="1491349079977">
    <contextId>globalroot-0</contextId>
    <layer3Sections>
        <section id="b33fc6fa-1f26-4103-aa1f-977c2036bb50" name="Universal Rules" generationNumber="1491353815142" timestamp="1491353815142" managedBy="universalroot-0" type="LAYER3">
            <rule id="2147483802" disabled="false" logged="false" managedBy="universalroot-0">
                <action>allow</action>
                <appliedToList>
                    <appliedTo>
                        <name>DISTRIBUTED_FIREWALL</name>
                        <value>DISTRIBUTED_FIREWALL</value>
                        <type>DISTRIBUTED_FIREWALL</type>
                        <isValid>true</isValid>
                    </appliedTo>
                </appliedToList>
                <sectionId>b33fc6fa-1f26-4103-aa1f-977c2036bb50</sectionId>
                <sources excluded="false">
                    <source>
                        <name>SR5430988-SRC-01</name>
                        <value>ipset-a65057ea-e6ee-418b-b123-5562bafbd783</value>
                        <type>IPSet</type>
                        <isValid>true</isValid>
                    </source>
                </sources>
                <direction>inout</direction>
                <packetType>any</packetType>
            </rule>
        </section>
    </layer3Sections>
    <generationNumber>1491353815142</generationNumber>
</filteredfirewallConfiguration>
```

**Comment by nmbradford** [comment]
Ah - I know why you said this would never have worked - but actually, it did.  NSX 6.1 -> NSX 6.2 changed the element name for returned rules which is why this test is in there, but it was written before we changed the module to run under strict mode where this now throws.  Thanks will review PR shortly

**Comment by dcoghlan** [comment]
Fixed in #174
